mobile/src/components/course/components/choose-mentor/choose-mentor.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { FC, ReactElement, useEffect } from 'react';
+
+import { AppColor, DataStatus } from '~/common/enums/enums';
+import {
+  FlatList,
+  RefreshControl,
+  Search,
+  Spinner,
+  Text,
+  View,
+} from '~/components/common/common';
+import { useAppDispatch, useAppSelector, useState } from '~/hooks/hooks';
+import { coursesActions } from '~/store/actions';
+
+import { MentorCard } from './components/components';
+import { styles } from './styles';
+
+const ChooseMentor: FC = () => {
+  const [searchValue, setSearchValue] = useState('');
+  const [isLoading] = useState(false);
```

Comment:
It`s redundant, bcos it won`t be changed

Reply:
You mean  `const [isLoading] = useState(false);` ?  I'm using that for RefreshControl

Reply:
But you don't control that value/ It will always be 'false'.

Reply:
I didn't pass this value in categories list and it works fine, so I think you can remove it at all